mtp_api/apps/mtp_auth/fixtures/initial_groups.json

```diff
@@ -131,7 +131,10 @@
       ["change_checkautoacceptrule", "security", "checkautoacceptrule"],
       ["add_checkautoacceptrulestate", "security", "checkautoacceptrulestate"],
       ["view_checkautoacceptrulestate", "security", "checkautoacceptrulestate"],
-      ["change_checkautoacceptrulestate", "security", "checkautoacceptrulestate"]
+      ["change_checkautoacceptrulestate", "security", "checkautoacceptrulestate"],
+      ["add_monitoredpartialemailaddress", "security", "monitoredpartialemailaddress"],
+      ["delete_monitoredpartialemailaddress", "security", "monitoredpartialemailaddress"],
+      ["view_monitoredpartialemailaddress", "security", "monitoredpartialemailaddress"]
```

Comment:
could be good to add the trailing comma there :)

Reply:
i’d LOVE to but it’s json :(

Reply:
ahaha darn it, every single time. Ignore 😝